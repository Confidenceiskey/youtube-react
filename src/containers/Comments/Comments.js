import React, { Component } from 'react';
import './Comments.scss';
import AddComment from './AddComment/AddComment';
import Comment from './Comment/Comment';
import CommentsHeader from './CommentsHeader/CommentsHeader';

class Comments extends Component {
  render() {
    return (
      <div className='comments-header'>
        <CommentsHeader amountComments={this.props.amountComments} />
        <AddComment />
        <Comment />
        <Comment />
        <Comment />
      </div>
    );
  }
}

export default Comments;
