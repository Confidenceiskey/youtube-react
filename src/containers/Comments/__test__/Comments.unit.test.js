import React from 'react';
import { shallow } from 'enzyme';
import Comments from '../Comments';

describe('Comments', () => {
  it('renders Comments without props', () => {
    const wrapper = shallow(
      <Comments />
    );
    expect(wrapper).toMatchSnapshot();
  });

  it('renders Comments with props', () => {
    const wrapper = shallow(
      <Comments amountComments={112499} />
    );
    expect(wrapper).toMatchSnapshot();
  });
});