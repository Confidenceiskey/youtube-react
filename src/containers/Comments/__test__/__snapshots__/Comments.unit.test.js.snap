// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comments renders Comments with props 1`] = `
<div
  className="comments-header"
>
  <CommentsHeader
    amountComments={112499}
  />
  <AddComment />
  <Comment />
  <Comment />
  <Comment />
</div>
`;

exports[`Comments renders Comments without props 1`] = `
<div
  className="comments-header"
>
  <CommentsHeader />
  <AddComment />
  <Comment />
  <Comment />
  <Comment />
</div>
`;
